app/models/damage.rb

```diff
@@ -0,0 +1,24 @@
+class Damage < ActiveRecord::Base
```

Comment:
There are two things with this model that I'm skeptical / unclear about:

1. Whether `Damage` is a great name - damage is more of a substantive, so saying "a damage" doesn't really work;
2. I don't understand what a `damage_fixed_reservation` or `damage_issued_reservation` are conceptually. I understand that damage might occur during a certain reservation, but how would damage be fixed during a reservation?